dev/AppNotifications/AppNotificationBuilder/AppNotificationButton.cpp

```diff
@@ -121,17 +121,42 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
     {
         auto logTelemetry{ AppNotificationBuilderTelemetry::ButtonToString::Start(g_telemetryHelper) };
 
-        std::wstring xmlResult{ wil::str_printf<std::wstring>(L"<action content='%ls'%ls%ls%ls%ls%ls%ls/>",
+        std::wstring xmlResult{ wil::str_printf<std::wstring>(L"<action content='%ls'%ls%ls%ls%ls%ls%ls%ls/>",
             m_content.c_str(),
             GetActivationArguments().c_str(),
             m_useContextMenuPlacement ? L" placement='contextMenu'" : L"",
             m_iconUri ? wil::str_printf<std::wstring>(L" imageUri='%ls'", m_iconUri.ToString().c_str()).c_str() : L"",
             !m_inputId.empty() ? wil::str_printf<std::wstring>(L" hint-inputId='%ls'", m_inputId.c_str()).c_str() : L"",
             GetButtonStyle().c_str(),
-            !m_toolTip.empty() ? wil::str_printf<std::wstring>(L" hint-toolTip='%ls'", m_toolTip.c_str()).c_str() : L"") };
+            !m_toolTip.empty() ? wil::str_printf<std::wstring>(L" hint-toolTip='%ls'", m_toolTip.c_str()).c_str() : L"",
```

Comment:
Is `m_toolTip` under the app's control? It should probably go through a sanitization pass to replace quotes & double-quotes so an app doesn't accidentally break the markup here...

Reply:
@jonwis 
Yes can set the tool tip as below, 
<img width="742" alt="image" src="https://github.com/user-attachments/assets/23b879dd-9c5c-4d91-9607-871da1fa214a">

Are you referring to the quotes & double-quotes that can be added as part of the string?
Yes, we should escape those unique chars but same holds good for other strings we accept as well rt ? I can raise sperate bug for this , please let me know.

Reply:
Anything that's under the app's control should be escaped here.  That includes special characters like quote/doublequote, <, >, etc.

Reply:
This "m_toolTip" is not added as part of this feature its existing.
I see those special characters (<,>,',") already escaped in the "set" method below.

![image](https://github.com/user-attachments/assets/0324ad6a-43ff-42cb-a62d-9bf62b5ef92c)